pkg/xds/envoy/routes/route_configuration_builder.go

```diff
@@ -54,6 +54,15 @@ func (b *RouteConfigurationBuilder) Build() (envoy.NamedResource, error) {
 	}
 }
 
+func (b *RouteConfigurationBuilder) MustBuild() envoy.NamedResource {
```

Comment:
is it used anywhere? goland says it's not

Reply:
removed